model/result/operations/replace/Service/ReplaceResultParserInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoLtiConsumer\model\result\operations\replace\Service;
+
+use oat\taoLtiConsumer\model\result\operations\replace\ReplaceResultOperationRequest;
+use Psr\Http\Message\ServerRequestInterface;
+
+interface ReplaceResultParserInterface
+{
+    public const REPLACE_RESULT_ROLE = 'https://purl.imsglobal.org/spec/lti-bo/scope/basicoutcome';
```

Comment:
are you sure that the spec on IMS this values is also used in LTI 1.3, 
because : https://www.imsglobal.org/forum/learning-tools-interoperability-public-forum/198081 explains that this is for LTI 1p1

Reply:
This scope is compliant with 
https://www.imsglobal.org/spec/lti-bo/v1p1#service-access-and-scope